lib/lightning_web/live/workflow_live/editor_pane.ex

```diff
@@ -78,6 +78,8 @@ defmodule LightningWeb.WorkflowLive.EditorPane do
     {:noreply, socket}
   end
 
+  # NOTE: This is dead code and should probably be removed. All events are
```

Comment:
Maybe we should remove it in this work, what do you think ?

Reply:
I don't feel confident enough to remove it at this time. I was thinking of removing it as a follow-up when everything is tested and works fine